config/class-sync.php

```diff
@@ -57,7 +57,7 @@ public function filter_cron_schedules( $schedule ) {
 
 		$schedule[ self::CRON_INTERVAL_NAME ] = [
 			'interval' => self::CRON_INTERVAL,
-			'display' => __( 'VIP Config Sync cron interval' ),
+			'display' => __( 'Custom interval for VIP Config Sync. Currently set to 5 mins' ),
```

Comment:
```suggestion
			'display' => __( `Custom interval for VIP Config Sync. Currently set to ${ CRON_INTERVAL } seconds` ),
```

Or you could extract 5 to its own variable